### PR TITLE
Bench test clean

### PR DESCRIPTION
**Pull request by dron0gus**
Do not pass strings as arguments where possible
Do not use strings as temp variables to store floats

### CODE REVIEW

File: firmware/controllers/bench_test.cpp [hunk 1]
```diff
@@ -129,153 +126,153 @@ static int count;
...
 
-static void pinbench(const char *delayStr, const char *onTimeStr, const char *offTimeStr, const char *countStr,
-		OutputPin* pinParam, brain_pin_e brainPinParam) {
-	delayMs = atoff(delayStr);
-	onTime = atoff(onTimeStr);
-	offTime = atoff(offTimeStr);
-	count = atoi(countStr);
-
-	brainPin = brainPinParam;
+static void pinbench(float dly, float on, float off, int cnt,
```
Comment (mck1117) [inline firmware/controllers/bench_test.cpp:129]: no contractions of short words please

Comment (dron0gus) [reply firmware/controllers/bench_test.cpp:129]: All self-explaining names are used by static globals

Comment (mck1117) [reply firmware/controllers/bench_test.cpp]:
```
struct state
{
    int count;
    float on;
    .......
};
state benchState;



{
    benchState.count = count;
    benchState.on = on;
     ...........
}
```

Comment (dron0gus) [reply firmware/controllers/bench_test.cpp]: `state` can be merged to `BenchController`. But lets do it in next iteration.

**Pull request merged**